### PR TITLE
feat: loading state for quick filter genders

### PR DESCRIPTION
**Pull request by egoroza**
This PR adds a loading state to the gender filter pills using placeholders when all of our filters are on one line. This ensures that the location selector and select sort dropdowns don't shift over dramatically once the data loads which can be confusing and jarring for the ux.

Screen Shot 2023-03-01 at 11 45 56 AM
Screen Shot 2023-03-01 at 11 46 09 AM

### CODE REVIEW

File: src/components/LoansByCategory/QuickFilters/QuickFilters.vue [hunk 1]
```diff
@@ -403,6 +412,20 @@ export default {
...
+		@apply tw-rounded !important;
+	}
+
+	.placeholder {
+		display: none;
+	}
+
+	@media screen and (min-width: 600px) {
+		.placeholder {
+			display: flex;
```
Comment (dyersituations) [inline src/components/LoansByCategory/QuickFilters/QuickFilters.vue:425]: These two displays could be converted to TW.

File: src/components/LoansByCategory/QuickFilters/QuickFilters.vue [hunk 2]
```diff
@@ -30,13 +30,20 @@
...
+			<div v-if="!removeGenderDropdown && !withCategories" class="tw-flex tw-gap-1 tw-overflow-x-auto tw-pb-1">
 				<filter-pills
 					:filters-loaded="filtersLoaded"
 					:options="filterOptions.gender"
 					:selected-values="selectedGenders"
 					@update-values="updateGenders($event)"
 				/>
+				<div v-if="!filtersLoaded" class="tw-flex tw-gap-1 placeholder">
+					<kv-loading-placeholder
+						style="height: 2.75rem; width: 100px;"
```
Comment (dyersituations) [inline src/components/LoansByCategory/QuickFilters/QuickFilters.vue:42]: Looks like the loading placeholders are 1px shorter, and the `height` can be removed here to fix that.

File: src/components/LoansByCategory/QuickFilters/QuickFilters.vue (hunk 1 above)
Comment (egoroza) [reply src/components/LoansByCategory/QuickFilters/QuickFilters.vue:425]: I think the TW breakpoint is slightly different (734px) ?

File: src/components/LoansByCategory/QuickFilters/QuickFilters.vue (hunk 2 above)
Comment (dyersituations) [reply src/components/LoansByCategory/QuickFilters/QuickFilters.vue]: Done.

File: src/components/LoansByCategory/QuickFilters/QuickFilters.vue (hunk 1 above)
Comment (dyersituations) [reply src/components/LoansByCategory/QuickFilters/QuickFilters.vue:425]: Good call, leaving as is for now.

**Pull request merged**